assets/configuration/dune

```diff
@@ -1,4 +1,4 @@
 (install
  (section bin)
  (package Oni2)
- (files setup.json configuration.json keybindings.json))
```

Comment:
Thanks for handling this for keybindings.json as well - it's great to not have to worry about these files